frontend/src/components/questions/TextQuestion.vue

```diff
@@ -1,5 +1,39 @@
+<script setup lang="ts">
+import { ref, computed } from 'vue';
+import type { Quizoot } from '@interfaces/quizoot';
+
+interface TextQuestionProps {
+    spec: Quizoot.TextQuestion;
+}
+
+const props = defineProps<TextQuestionProps>();
+
+const answer = ref('');
+
+const placeholder = computed(
+    () => props.spec.placeholder ?? 'Type your answer here...'
+);
+</script>
+
 <template>
-    <div>Hello <code>TextQuestion</code></div>
+    <div class="text-question">
+        <input :placeholder="placeholder" v-model="answer" />
```

Comment:
* Souldn't you add the attribute `minlength=1` or `required` ?

* Also, don't we need here a `textarea` instead of `input` ?